corehq/apps/reports/generic.py

```diff
@@ -674,12 +674,12 @@ def filters_response(self):
 
     @property
     @request_cache()
-    def json_response(self):
+    def JsonResponse(self):
```

Comment:
I think this change must have been unintentional, right?

Reply:
As per usual @emord seems to have beaten me to it: https://github.com/dimagi/commcare-hq/pull/24090